src/main/resources/application.yaml

```diff
@@ -576,3 +576,9 @@ bulk-action:
   page-size: ${BULK_ACTION_BATCH_SIZE_MAX:50}
 
 SERVICE_AUTH_MICROSERVICE: nfdiv_case_api
+
+feign:
+  client:
+    config:
+      connectTimeout: 10000000
+      readTimeout: 20000000
```

Comment:
Have you tried reducing these? 10000000 ms would mean locking up a web thread for nearly 3 hours while it waits for a connection

Reply:
This was just a random attempt to fix something. I will revert this change as this is not required 😃